features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/public/locales/en.json

```diff
@@ -1,87 +1,97 @@
+  "blocked": "Blocked",
+  "blocked.production.only": "Blocked Production Only",
+  "business.plans": "Business Plans",
+  "by": "by",
+  "by.swagger.url": "By OpenAPI url",
+  "bytes": "bytes",
+  "cancel.btn": "CANCEL",
+  "context": "Context",
+  "create": "Create",
+  "create.a.document.from.file.or.url": "Create a document from file or URL.",
+  "create.a.document.from.file.or.url.decription": "Create a new document using a file or URL for this api.",
+  "create.a.new.inline.document.for.api": "Create a new inline document for this api.",
+  "create.an.api": "Create an API",
+  "create.an.endpoint": "Create An Endpoint",
+  "create.an.inline.document": "Create an inline document",
+  "create.api": "Create API",
+  "create.btn": "CREATE",
   "create.new.api": "Create New API",
+  "create.new.api.swagger": "Create New API - ",
   "create.new.rest.api": "Create New REST API",
   "create.new.websocket.api": "Create New WebSocket API",
-  "create.new.api.swagger": "Create New API - ",
-  "swagger.file.upload": "OpenAPI file upload",
-  "by.swagger.url": "By OpenAPI url",
-  "try.dropping.some.files.here.or.click.to.select.files.to.upload": "Try dropping some files here, or click to select files to upload.",
-  "file": "File",
-  "url": "URL",
-  "uploaded.files": "Uploaded files",
+  "create.scope.description": "Scopes enable fine-grained access control to API resources based on user roles. You define scopes to an API's resources. When a user invokes the API, his/her OAuth 2 bearer token cannot grant access to any API resource beyond its associated scopes.",
+  "create.scopes": "Create Scopes",
+  "define.global.endpoint": "Define Global Endpoint",
   "deploy.sample.api": "Deploy Sample API",
-  "use.an.existing.api.endpoint.or.the.api.swagger.definition.to.create.an.api": "Use an existing API endpoint or the API OpenAPI definition to create an API",
-  "i.have.a.soap.endpoint": "I Have a SOAP Endpoint",
-  "use.an.existing.soap.or.import.the.wsdl": "Use an existing SOAP or Import the WSDL",
-  "use.an.existing.soap.endpoint.to.create.a.managed.api.import.the.wsdl.of.the.soap.service": "Use an existing SOAP endpoint to create a managed API. Import the WSDL of the SOAP service",
   "design.a.new.rest.api": "Design a New REST API",
-  "i.have.an.existing.api": "I Have an Existing REST API",
-  "use.an.existing.rest.endpoint.or.Swagger.definition": "Use an existing REST endpoint or OpenAPI definition",
   "design.a.new.rest.api.using.wsdl": "Design a new REST API using WSDL",
-  "select.wsdl": "Select WSDL",
   "design.and.prototype.a.new.rest.api": "Design and prototype a new REST API",
-  "fill.the.mandatory.fields": "Fill the mandatory fields (Name, Version, Context) and create the API. Configure advanced configurations later.",
-  "design.new.websocket.api": "Design New WebSocket API",
   "design.and.prototype.a.new.websocket.api": "Design and prototype a new WebSocket API",
+  "design.new.websocket.api": "Design New WebSocket API",
```

Comment:
any reason why the existing keys removed and added again?

Reply:
Now they're sorted in the alphabetical order which will help us to avoid adding duplicate keys.